pom.xml

```diff
@@ -202,6 +202,28 @@
                 <artifactId>struts-taglib</artifactId>
                 <version>1.3.10</version>
             </dependency>
+            <dependency>
+                <groupId>com.mockrunner</groupId>
+                <artifactId>mockrunner-core</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.mockrunner</groupId>
+                <artifactId>mockrunner-jms</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.mockrunner</groupId>
+                <artifactId>mockrunner-servlet</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.mockrunner</groupId>
+                <artifactId>mockrunner-core</artifactId>
```

Comment:
mockrunner-core is defined twice in dependency management. That won't work. We must have only one, and then perhaps this specific type:xml, classifier:features can be overrides in the mock runner-servlet module?

Reply:
You're right that it won't work, but for a different reason.

The first occurrence is the MockRunner Core (JAR), the second one should be the Features classifier of type XML. But you can't "override" the classifier or the type in the child module - everything except version and scope must be equal.

So I've updated the child module to also mention we're looking for the Features classifier of type XML, and now it should resolve to the correct version as declared in the parent Dependency Management.